src/main/java/SwitchSearchListPage.java

```diff
@@ -0,0 +1,17 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SwitchSearchListPage {
+
+    WebDriver driver;
+    public SwitchSearchListPage(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public SwitchNeonRedBluePage OpenSwitchNeonRedBluePage(){
+
+        driver.findElement(By.cssSelector("[data-cel-widget='search_result_1'] span.a-size-base-plus")).click();
```

Comment:
Can you move the selectors as class members variables? every selector in the page.